orchestrator-salt/src/main/resources/salt/salt/metering/scripts/metering_package_manager.sh

```diff
@@ -0,0 +1,316 @@
+#!/bin/sh
+
+METERING_PACKAGE_NAME="thunderhead-metering-heartbeat-application"
+METERING_FILE_NAME="$METERING_PACKAGE_NAME-0.1-SNAPSHOT.x86_64.rpm"
+: ${METERING_SERVICE_DELIVERY_URL:="https://cloudera-service-delivery-cache.s3.amazonaws.com/$METERING_PACKAGE_NAME/clients/$METERING_FILE_NAME"}
+# todo: provide available archive.cloudera.com rpm release location
+: ${METERING_ARCHIVE_URL:="https://cloudera-service-delivery-cache.s3.amazonaws.com/$METERING_PACKAGE_NAME/clients/$METERING_FILE_NAME"}
```

Comment:
shouldn't be merged without archive url

Reply:
i updated to won't check both if those are the same

Reply:
also it seems the new archive.cloudera.com address will be used for snapshots (although that is uploaded by manually at the moment), but point is i can use that in this script